app/views/attractions/create.js.erb

```diff
@@ -0,0 +1 @@
+console.log('successful create')
```

Comment:
Since not all browsers support `console.log`, you should really not use it in production.  

If you must, make you sure define `console.log` for browsers that don't have it: `console = console || {}; console.log = console.log || function() {};`.